spa/src/components/account/Profile/Profile.js

```diff
@@ -0,0 +1,74 @@
+import { useReducer } from 'react';
+import { useHistory } from 'react-router-dom';
+import * as S from './Profile.styled';
+import * as A from 'components/account/Account.styled';
+
+// AJAX
+import axios from 'ajax/axios';
+import { CUSTOMIZE_ACCOUNT_ENDPOINT } from 'ajax/endpoints';
+
+// State management
+import fetchReducer, { initialState, FETCH_START, FETCH_FAILURE } from 'state/fetch-reducer';
+import { useUserContext } from 'components/UserContext';
+
+import useModal from 'hooks/useModal';
+
+// Analytics
+import { useConfigureAnalytics } from 'components/analytics';
+
+import ProfileForm from './ProfileForm';
+
+function Profile() {
+  const { open, handleClose } = useModal(true);
+  const history = useHistory();
+  const [profileState, dispatch] = useReducer(fetchReducer, initialState);
+  const { user } = useUserContext();
+  useConfigureAnalytics();
+
+  const onProfileSubmit = async (formData) => {
+    dispatch({ type: FETCH_START });
+
+    try {
+      const { data, status } = await axios.patch(`users/${user.id}/${CUSTOMIZE_ACCOUNT_ENDPOINT}`, {
+        first_name: formData.firstName,
+        last_name: formData.lastName,
+        job_title: formData.jobTitle,
+        organization_name: formData.companyName,
+        organization_tax_status: formData.companyTaxStatus
+      });
+
+      if (status === 204) {
+        history.push('/');
+      } else {
+        dispatch({ type: FETCH_FAILURE, payload: data });
+      }
+    } catch (e) {
+      dispatch({ type: FETCH_FAILURE, payload: e?.response?.data });
+    }
+  };
+
+  const formSubmitErrors =
+    Array.isArray(profileState?.errors) && profileState?.errors.length === 0 ? null : 'An Error Occurred';
```

Comment:
Probably could change it to just:
```suggestion
    !profileState?.errors?.length && 'An Error Occurred';
```

Reply:
I think
```
profileState?.errors?.length && 'An Error Occurred';
```

(e.g. no `!` in front?)

Reply:
Yes 👍

Reply:
Forgot to actually address this, did so in e823757195c65f58aa97f8e828490f29615d8652